<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTaskStandardUserAccountDialog</class>
 <widget class="QDialog" name="ViewTaskStandardUserAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TRMS - Task Details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/images/TRMS-Logo-Without-Text.ico</normaloff>:/images/TRMS-Logo-Without-Text.ico</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(31, 192, 196);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="taskTitle_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TASK TITLE</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="taskDescription_plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myriad Pro Light</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="plainText">
     <string>TASK DESCRIPTION</string>
    </property>
   </widget>
   <widget class="QLabel" name="color_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="categoryName_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myriad Hebrew</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Category Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="taskLastEdited_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Task Last Edited:</string>
    </property>
   </widget>
   <widget class="QLabel" name="taskCreated_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Task Created:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editTaskDetails_pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 249, 249);</string>
    </property>
    <property name="text">
     <string>EDIT TASK DETAILS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeTask_pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 249, 249);</string>
    </property>
    <property name="text">
     <string>REMOVE TASK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setReminder_pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semilight</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 249, 249);</string>
    </property>
    <property name="text">
     <string>SET REMINDER</string>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/images/Reminders.png</normaloff>:/images/Reminders.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="taskCreatedDateTime_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>datetime</string>
    </property>
   </widget>
   <widget class="QLabel" name="taskLastEditedDateTime_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>datetime</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
